### PR TITLE
Providing Typedoc references for the Chip topic.

### PR DESCRIPTION
**Pull request by MartinKamenov**

### CODE REVIEW

**Comment by yoshimis** [comment]
Reviewed and no localization necessary for this PR. Please add a label "pending-localization" back if any new changes added.

File: en/components/chip.md [hunk 1]
```diff
@@ -165,13 +165,22 @@ The chip can be focused using the `Tab` key or by clicking on them. Chips can be
...
   - <kbd>SHIFT</kbd> + <kbd>RIGHT</kbd> - Moves the focused chip one position to the right
 
 - Keyboard controls when the remove button is focused:
 
-  - <kbd>SPACE</kbd> or <kbd>ENTER</kbd> Fires the `onRemove` output so the chip deletion can be handled manually
+  - <kbd>SPACE</kbd> or <kbd>ENTER</kbd> Fires the [`onRemove`]({environment:angularApiUrl}/classes/igxchipcomponent.html#onremove) output so the chip deletion can be handled manually
+
+### API
+
+* [IgxChipComponent]({environment:angularApiUrl}/classes/igxchipcomponent.html)
```
Comment (HristoP96) [inline en/components/chip.md]: Provide after this line a Sassdoc reference to the igx-chip-theme as follows:
[IgxChipComponent Styles] ({environment:sassApiUrl}/index.html#function-igx-chip-theme)

File: jp/components/chip.md [hunk 2]
```diff
@@ -166,13 +166,22 @@ public chipsOrderChanged(event) {
...
   - <kbd>SHIFT</kbd> + <kbd>RIGHT</kbd> - フォーカスされたチップの位置を 1 つ右へ移動します。
 
 - 削除ボタンがフォーカスされた場合のキーボード コントロール:
 
-  - <kbd>SPACE</kbd> または <kbd>ENTER</kbd> チップの削除を手動的に処理するために `onRemove` 出力を発生します。
+  - <kbd>SPACE</kbd> または <kbd>ENTER</kbd> チップの削除を手動的に処理するために [`onRemove`]({environment:angularApiUrl}/classes/igxchipcomponent.html#onremove) 出力を発生します。
+
+### API
+
+* [IgxChipComponent]({environment:angularApiUrl}/classes/igxchipcomponent.html)
```
Comment (HristoP96) [inline jp/components/chip.md]: Provide Sassdoc reference to the igx-chip-theme.

Comment (MartinKamenov) [reply jp/components/chip.md]: Please review the changes that were made.

**Pull request merged**